lms/djangoapps/mobile_api/social_facebook/friends/tests.py

```diff
@@ -0,0 +1,394 @@
+                                            }
+                                        })
+        # Set the interceptor for the first paged content 
+        httpretty.register_uri( httpretty.GET, 
+                                "https://graph.facebook.com/v2.2/me/friends/next_1",
+                                body=json.dumps({"data": [ {'name': self.USERNAME_2, 
+                                                            'id': self.FB_ID_2}], 
+                                                "paging": { 
+                                                    "next": "https://graph.facebook.com/v2.2/me/friends/next_2"
+                                                    }, 
+                                                "summary": {
+                                                    "total_count": 652
+                                                    }
+                                                }), 
+                                status=201)
+        # Set the interceptor for the last paged content
+        httpretty.register_uri( httpretty.GET, 
+                                "https://graph.facebook.com/v2.2/me/friends/next_2",
+                                body=json.dumps({"data": [ {'name' : self.USERNAME_3, 
+                                                            'id' : self.FB_ID_3}], 
+                                                "paging": {"previous": "https://graph.facebook.com/v2.2/10154805434030300/friends?limit=25&offset=25"}, 
+                                                "summary": {"total_count": 652}
+                                                }), 
+                                status=201)
+
+        url = reverse('friends-in-course', kwargs={"course_id": self.format_course_id()})
+        response = self.client.get(url, {   'format' : 'json', 
+                                            'oauth_token' : self._FB_USER_ACCESS_TOKEN})
+                
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('friends' in response.data)
+        # Assert that USERNAME_1 is returned
+        self.assertTrue('id' in response.data['friends'][0] and response.data['friends'][0]['id'] == self.FB_ID_1)
+        self.assertTrue('name' in response.data['friends'][0] and response.data['friends'][0]['name'] == self.USERNAME_1)
+        # Assert that USERNAME_2 is returned
+        self.assertTrue('id' in response.data['friends'][1] and response.data['friends'][1]['id'] == self.FB_ID_2)
+        self.assertTrue('name' in response.data['friends'][1] and response.data['friends'][1]['name'] == self.USERNAME_2)
+        # Assert that USERNAME_3 is returned
+        self.assertTrue('id' in response.data['friends'][2] and response.data['friends'][2]['id'] == self.FB_ID_3)
+        self.assertTrue('name' in response.data['friends'][2] and response.data['friends'][2]['name'] == self.USERNAME_3)
+
+
+    '''
+        Helper Functions 
+    '''
```

Comment:
All of these helper functions can be defined in a common test_utils method within the social_facebook folder and shared across the test files.  You can create a common SocialFacebookTestCase base class that implements these.